#request



---

# response


float64 x_current
float64 y_current
float64 z_current
